### PR TITLE
update: add permsMng contract

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: contracts/Entry.sol [hunk 1]
```diff
@@ -1,247 +1,73 @@
...
-        require(msg.sender == perm.executor, "Sender is not valid executor");
-        address ctrAddr = perm.ctrAddr;
+    function execute(uint256 id, bytes args) external {
+        PermsMng p = PermsMng(permsMng);
+        address ctrAddr = p.perms(id);
+        address govern = p.governs(id);
+        require(msg.sender == govern, "Specified Govern Required");
         assembly {
             let ptr := mload(40)
+            mstore(ptr, calldataload(0x64))
```
Comment (kaikai1024) [inline contracts/Entry.sol]: ??
should to delete it

File: contracts/Entry.sol [hunk 2]
```diff
@@ -1,247 +1,73 @@
...
             calldatacopy(ptr, 0x64, sub(calldatasize, 0x64))
             switch call(gas, ctrAddr, 0, ptr, 0x24, ptr, 0)
```
Comment (kaikai1024) [inline contracts/Entry.sol]: 0x24 -> sub(calldatasize, 0x64)

File: contracts/PermsMng.sol [hunk 3]
```diff
@@ -0,0 +1,87 @@
+pragma solidity 0.4.25;
+
+contract PermsMng {
+    // owner from entry contract
+    address owner;
+    // perm addresses
+    address[33] public perms;
```
Comment (kaikai1024) [inline contracts/PermsMng.sol]: why not `address[]`

Comment (Keith-CY) [reply contracts/PermsMng.sol]: it threw an error "Instruction is not supported."

**Pull request merged**